openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -454,11 +454,15 @@ def _is_cookie_present(response):
     )
 
 
-def _delete_cookie(response):
+def _delete_cookie(request, response):
     """
     Delete the cookie by setting the expiration to a date in the past,
     while maintaining the domain, secure, and httponly settings.
     """
+    log.warning(
+        "SafeSessionMiddleware is deleting session cookie for user %d",
```

Comment:
Nit: Use the same log prefix?  `SafeCookieData`

Reply:
Sure thing.